### PR TITLE
basic styling using styled-components setup

### PR DESCRIPTION
**Pull request by christopher-tucker**
@yarosz would you mind doing a quick code review?

The only real changes to review are in the .jsx files.
Also, I'm using styled-components for the first time, feel free to see what I did to set them up if you're interested! Let me know if you have questions about what I did, so far it has been fairly simple. 

Thanks!
-Chris

### CODE REVIEW

**Comment by yarosz** [comment]
@christopher-tucker Great, reviewing now...

File: src/components/App.jsx [hunk 1]
```diff
@@ -22,12 +28,17 @@ class App extends React.Component {
     });
```
Comment (yarosz) [inline src/components/App.jsx:28]: Your promise chain above (lines 23 & 26) should be indented inside your axios.get :)

File: src/components/PledgeList.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import SinglePledge from './SinglePledge.jsx';
+
+const PledgeList = (props) => {
+  const { pledges } = props;
```
Comment (yarosz) [inline src/components/PledgeList.jsx:5]: I'm curious - for my own understanding - how is the assignment on **line 5**  different from the assignment in **src/components/App.jsx** on **line 41**

Is this redundant or required to get the pledges passed into the PledgeList's props?

File: src/components/PledgeList.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <div className="PledgeListComponentDiv">
+      {pledges.map((pledge, index) => <SinglePledge pledge={pledge} key={index}/>)}
```
Comment (yarosz) [inline src/components/PledgeList.jsx:8]: Great job getting this **.map** so succinctly written!

File: src/components/SinglePledge.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const PledgeBox = styled.div`
+  text-align: center;
+  font-family: Ariel;
```
Comment (yarosz) [inline src/components/SinglePledge.jsx:6]:
```suggestion
  font-family: Arial;
```
Is this a typo?

File: src/components/App.jsx (hunk 1 above)
Comment (christopher-tucker) [reply src/components/App.jsx:28]: Ah thanks!

File: src/components/PledgeList.jsx (hunk 2 above)
Comment (christopher-tucker) [reply src/components/PledgeList.jsx:5]: On line 41 in App.jsx when I pass "pledges" into the PledgeList component, I'm passing that data in as props, so they appear in PledgeList.jsx inside the props object. 
The props object then looks something like this: 
props = {
    pledges: [....]
}
So inside PledgeList, I do a spread operator to extract out the variable I want from props. One thing to note is that doing a spread operator becomes a lot more useful when there are a bunch of props being sent down into a child component, where here we only have one being passed down.

I could have omitted line 5 here, and then on line 8, I could have said props.pledges.map(...etc.. ). It's kind of a style thing that helps me organize.

Comment (yarosz) [reply src/components/PledgeList.jsx:5]: Okay great. This makes sense - I see how it's very useful in SinglePledge component where you are managing 7 different props!

Thanks for the explanation @christopher-tucker

**Pull request merged**